src/app/auctioneer/auctioneer.service.ts

```diff
@@ -57,28 +57,27 @@ export class AuctioneerService {
                 'Authorization': 'Bearer ' + this.authService.getToken()
             })
         }
-        
+
         return this.httpClient.get<any>(`${environment.BASE_URL}/auctions`, httpOptions)
             .subscribe(response => {
                 if(response){
-                    
+
                     this.auctionItems = [];
                     response.result.forEach( (auction) => {
                         this.auctionItems.push(auction);
```

Comment:
With the change of AuctionItem becoming a class instead of an interface, the auto type casting of the response json into an AuctionItem object might need a change. It's fine for now since the class only has properties, but if we have to add any methods to the class, I don't believe they will work on the created objects. With an interface it was fine, but with a class, we might want to create the object properly with its constructor.